src/App.js

```diff
@@ -1,14 +1,35 @@
 import React from 'react';
 import './fonts.css';
 
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link
+} from "react-router-dom";
+
 import Noise from './components/noise';
 import Landing from './components/landing';
 
 const App = () => (
-  <div>
-    <Noise />
-    <Landing />
-  </div>
+  <Router>
+    
+
+    <Switch>
+      
+      <Route path='/registration'>
+        <div>
+          Registration
```

Comment:
Can you add the Typeform embed stuff here? The code is in #tech on discord, should be it's own component